.travis.yml

```diff
@@ -1,5 +1,15 @@
+sudo: false
+
 language: node_js
+
 node_js:
-  - "stable"
-  - "5"
-  - "4"
+  - 6
+  - 4
+
+cache:
+  directories:
+    - node_modules
+
+install:
+  - npm i -g npm@latest
```

Comment:
You didn't give answers to other questions to this PR. Seems like other changes are either unnecessary or you should tell more about them.

> sudo: false

As far as I understand TravisCI, this goes by default, isn't it?

> cache directories node_modules

Cool thing! But what if package.json dependencies change after PR is created?

> npm i -g npm@latest

This seems a bit unsafe to me. Each node comes with its own npm, why should we switch to latest node? Especially now when `mocha` is peerDependency of mocha-parallel-tests.